scripts/local/utils/bulk_ops_common.py

```diff
@@ -72,29 +52,79 @@ def get_local_argparser():
     local_argparser.add_argument(
         "-r",
         "--recursive",
-        required=True,
-        type=bool,
+        action='store_true',
         dest="recursive",
         help="Specify whether to process subdirectories recursively",
     )
 
     local_argparser.add_argument(
         "--trigger-size",
         default=200,
-        required=True,
         type=int,
         dest="trigger_size",
-        help="Specify minimum file size to trigger the hook.",
+        help="Specify minimum file size (KB) to trigger the hook.",
     )
+
     return local_argparser
 
 
+def convert_image(input_path, output_path, output_format):
+    """
+    Converts an image to the specified output format.
+    """
+    with Image.open(input_path) as img:
+        if output_format == 'JPG':
+            output_format = 'JPEG'
+        img.save(output_path, output_format)
+
+
+def convert_pdf_to_jpg(input_path, output_dir):
+    """
+    Converts a PDF to a series of JPG images, one per page.
+    """
+    pages = convert_from_path(input_path)
+    for i, page in enumerate(pages):
+        output_path = os.path.join(output_dir, f"page_{i + 1}.jpg")
+        page.save(output_path, 'JPEG')
+
+
+def bulk_convert(input_dir, output_dir, output_format, in_place=False):
+    """
+    Bulk converts images and PDFs to the specified format.
+    """
+    os.makedirs(output_dir, exist_ok=True)
+    extensions = ['png', 'jpg', 'jpeg', 'tiff', 'tif', 'pdf']
```

Comment:
Let's keep pdf handling in a separate script (check other comments)
```suggestion
    extensions = ['png', 'jpg', 'jpeg', 'tiff', 'tif']
```